### ISSUE TITLE
Solidify and promote travel fund

### ISSUE THREAD

**Issue by whit537** [sustainers/website#113]
Now that we have a travel assistance fund (#32; yay!), let's get the word out! But first, let's get our facts straight. ☺️ There was some confusion between @piamancini @jdorfman and myself on Slack earlier today about how much funding is available and which organizations are providing it. Facts as I have them:

1. My understanding from @BenJam on last week's call was that Sloan has allocated up to $20,000. Yes, @BenJam?
1. Per @piamancini in Slack, "the Core Infrastructure Initiative from Linux F. emailed that they want to give a couple of thousand on travel grants"
1. I've heard us half-suggesting that general sponsorship funds could be used for travel assistance as well. We have $5,000+ in OC right now. Is that all available for travel? If not let's make a separate ticket to budget.

This looks to me like Sloan is the anchor, with CII and other general sponsors chipping in additional. Am I reading this right? If so, can we run the whole thing through Sloan? Can CII donate $2,000 to Sloan rather than having to run "some" (which?) applicants through two separate application processes? Fewer UX bifurcations, please (#112). ;-)

**Comment by whit537** [comment]
@piamancini reports on private weekly call that flow of funds is Sloan &rarr; Aspiration, with distribution to individual attendees from there. We have 13 applications for travel so far, 10 from US and 3 from Australia/New Zealand. @piamancini and @jerodsanto are not on the list, we are thinking perhaps we use Linux travel funds for Sustain organizers.

**Comment by whit537** [comment]
We have 20+ applications, with four days left to apply. @BenJam and @piamancini are in charge of deciding how to distribute funds among applicants.

**Comment by whit537** [comment]
We've received 20 applications, and @BenJam and @piamancini have selected 15 for funding. Next step is to solidify w/ Sloan. We are on track to announce decisions on June 2 as planned.

Great work @BenJam @piamancini!

**Comment by whit537** [comment]
Can we goose the travel fund with general funds? I'm [seeing](https://opencollective.com/sustainoss) that we have $6,514.33 available right now. How far would that take us in covering the other five folks?

**Comment by BenJam** [comment]
> Can we goose the travel fund with general funds

Not sure what the goal is there. We've req. 20kUSD on an estimated cost of ~19,500USD for travel grant applications, so there's little wiggle room. 

Accounting wise: Aspiration is the grantee for the Sloan travel grant, while OC are holding the donations. Not sure who is the XO on deciding how donations are spent but it's Aspiration's call on the travel grant side.

**Comment by whit537** [comment]
Okay. It also seems likely that folks might end up over budget once they actually buy tickets, etc. Good to have some reserves to handle that as need arises.

**Comment by piamancini** [comment]
update from @BenJam: decision should come today form Sloan. I will notify successful peeps.

**Comment by whit537** [comment]
Per private weekly call, decisions have been made. We've notified everyone, both accepted and denied. We accepted 14 people, three have already said they can't make it because we're too close to the event. That takes some of the potential budget pressure off so we should be in good shape!

I am closing this since it's basically details from here that are between @BenJam and Aspiration.

Great job @BenJam @piamancini Sloan et al.! Such a huge contribution to the event! 🚀 ✨
